test/integration/profile_security_question_test.go

```diff
@@ -0,0 +1,20 @@
+package integration
+
+import (
+    "context"
+    "testing"
+
+    "github.com/stretchr/testify/require"
+)
+
+func TestSecurityQuestions_List(t *testing.T) {
+    client, teardown := createTestClient(t, "fixtures/TestSecurityQuestions_List")
```

Comment:
It might be good to add a warning here to let devs know this test could potentially store sensitive data in fixtures:

```go
func warnSensitiveTest(t *testing.T) {
	if testingMode == recorder.ModeReplaying {
		return
	}

	slog.Warn(
		fmt.Sprintf(
			"Test %s is a sensitive test. Ensure you validate and sanitize "+
				"its generated test fixtures before pushing.",
			t.Name(),
		),
	)
}

// ...

func TestSecurityQuestions_List(t *testing.T) {
	warnSensitiveTest(t)

	# ...
}